src/test/java/org/mondemand/tests/ClientTest.java

```diff
@@ -319,6 +319,8 @@ public void testSamplesMessages() throws Exception {
     Client client = createClientNoTransports();
     LWESTransport localLwesTransport = new LWESTransport(InetAddress.getLocalHost(), 9292, null);
     client.addTransport(localLwesTransport);
+    // add a second transport to test stats sample resets
+    client.addTransport(localLwesTransport);
```

Comment:
adding a second transport resulted in this test failing prior to these changes...